### DISCUSSION TITLE
Import Euler angles  from txtfile

### DISCUSSION THREAD

**Discussion by psaidi** [mtex-toolbox/mtex#1016] [Ask Anything] [answered]
I am trying to import Euler angles from a text file in the form of three columns (Import EBSD data). What I see in the wizard is three columns as I expected and then I specify the data type of each column (phi1, Phi and phi2), degree and passive rotation --> finish,  but the error dialog says could not detect file format. please help me to learn what I am missing here.  I attach a sample of the data set. 

Thank you for your help
Peyman
[test.txt](https://github.com/mtex-toolbox/mtex/files/6440499/test.txt)

**Comment by kilir** [comment, marked as answer]
Hi,
EBSD data expects also x,y coordinates. Please also have a look in the Discussions here, e.g. https://github.com/mtex-toolbox/mtex/discussions/1010.
Hope that helps.
Cheers,
Rüdiger

**Reply by psaidi** [reply]
thank you. I will try with x and y. 

cheers, Peyman

**Reply by stargazingdays** [reply]
Thanks Rüdiger. That works out for me.
